DOC: Add `InnerEye/Azure` docstrings to RTD (#788)
microsoft/InnerEye-DeepLearning: docs/source/rst/api/Azure/azure.rst

Azure

> =================

Reviewer: ```suggestion
=====
```

> Below you will find the docstrings of all resources found in `InnerEye/Azure <https://github.com/microsoft/InnerEye-DeepLearning/tree/main/InnerEye/Azure>`_.

Reviewer: ```suggestion
Below you will find the documentation for `InnerEye/Azure <https://github.com/microsoft/InnerEye-DeepLearning/tree/main/InnerEye/Azure>`_.
```

Runner
---------------

.. automodule:: InnerEye.Azure.azure_runner

Configuration
---------------

.. automodule:: InnerEye.Azure.azure_config

Utils

> -------

Reviewer: Warnings are raised if the underlines and the titles have different lengths.
